test/src/test/testtool/CorrelationMessageId/common.properties

```diff
@@ -9,6 +9,9 @@ java.CorrelationMessageIdXPath.serviceName=ibis4test-CorrelationMessageIdXPath
 ws.CorrelationMessageIdXPath.className=nl.nn.adapterframework.http.WebServiceSender
 ws.CorrelationMessageIdXPath.url=${web.protocol}://${web.host}:${web.port}${web.contextpath}/servlet/rpcrouter
 ws.CorrelationMessageIdXPath.serviceNamespace=CorrelationMessageIdXPath2
+ws.CorrelationMessageIdXPath.authAlias=framework.api.user.alias
+ws.CorrelationMessageIdXPath.allowSelfSignedCertificates=true
+ws.CorrelationMessageIdXPath.verifyHostname=false
```

Comment:
Could you make `allowSelfSignedCertificates=true` and `verifyHostname=false` the default for Larva?